### PR TITLE
Add missing types

### PR DESCRIPTION
**Pull request by gbaranski**

### CODE REVIEW

File: src/mixable.ts [hunk 1]
```diff
@@ -10,7 +10,12 @@ export default class Mixable {
 
 export interface MixResult {
     results: Result[]
+    type: MixableType,
```
Comment (stanbar) [inline src/mixable.ts]: `MixableType` is defined for input Mixeable types, not `MixResult` type. When you mix Premix with Nicoshot what will be the type ? When you mix Premix with Premix what will be type ? When you mix Liquid with Nicoshot what will be the type ? When you mix Premix with Premix what will be the type ? etc...

File: src/liquid.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ export class Result {
   constructor(
     readonly name: string,
     readonly percentage: number,
-    readonly ml: number,
+    readonly amount: number,
```
Comment (stanbar) [inline src/liquid.ts]: This change will break a lot of code, whole Liquid Calculator expect `.ml`, not `.amount`. Do you want to fix that code ? I have better suggestion, let's create another type `MixableResult` that will be used only in mixer. Then you can modify whatever you want ok ?

**Review by stanbar — CHANGES_REQUESTED** [review]

File: src/liquid.ts (hunk 2 above)
Comment (gbaranski) [reply src/liquid.ts]: sure
@stasbar

**Comment by stanbar** [comment]
Good job

**Pull request merged**